youtube_dl/extractor/adobepass.py

```diff
@@ -325,6 +320,11 @@
     'col080': {
         'name': 'Columbus Telephone'
     },
+    'Comcast_SSO': {
+        'name': 'Comcast XFINITY',
+        'username_field': 'user',
+        'password_field': 'passwd',
+    },
```

Comment:
See if my changes are suitable.  (Of course, there's still the matter of "Unable to extract post url," if there's any way someone could give me suggestions to try.)